gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=4.0.2
```

Comment:
I'm bumping the minor version to highlight the behavioural (non breaking) change for our timeouts management